packages/frontend/src/views/CalendarView/CalendarView.module.css

```diff
@@ -0,0 +1,51 @@
+.sidebarHeadings {
+  font-size: var(--flocker-font-size-medium);
+  font-weight: var(--flocker-font-weight-light);
+  margin-top: 16px;
+}
+
+.timeMatcher {
+  width: 60vw;
+  height: 70vh;
+}
+
+h2 {
+  font-size: var(--flocker-font-size-body);
+  font-weight: var(--flocker-font-weight-light);
+  padding: 16px 0 8px 0;
+}
+
+h3 {
+  font-size: var(--flocker-font-size-small);
+  font-weight: var(--flocker-font-weight-light);
+  padding: 0 10px 0 10px;
+}
+
+.legendKeys {
+  display: flex;
+  flex-direction: row;
```

Comment:
Isn't `row` the default `flex-direction`?

Reply:
:0

Reply:
Changed